adoc/CaaSP_DH2X_Install_Guide.adoc

```diff
@@ -559,61 +602,74 @@ PORTUS_PASSWORD=XXXXXXXX
     root /srv/Portus/public;
 ----
 
-Now, you can pull the latest *docker-compose.yml*.
-
+. Download the latest `docker-compose.yml` for Portus:
++
 ----
 # rm docker-compose.*
 # wget https://gist.githubusercontent.com/Patazerty/d05652294d5874eddf192c9b633751ee/raw/6bf4ac6ba14192a1fe5c337494ab213200dd076e/docker-compose.yml
 ----
 
-To avoid dealing with Docker insecure registry configuration add SSL to your setup.
-
+. To secure your Docker Registry configuration, add TLS to your setup:
++
 ----
 echo "subjectAltName = DNS:portus-dr.suse-sap.net" > extfile.cnf
+
 openssl genrsa -out secrets/rootca.key 2048
-openssl req -x509 -new -nodes -key secrets/rootca.key -subj "/C=FR/ST=FR/O=SUSE"  -sha256 -days 1024 -out secrets/rootca.crt
+
+openssl req -x509 -new -nodes -key secrets/rootca.key -subj "/C=FR/ST=FR/O=SUSE" \
+  -sha256 -days 1024 -out secrets/rootca.crt
+
 openssl genrsa -out secrets/portus.key 2048
-openssl req -new -key secrets/portus.key -out secrets/portus.csr -subj "/C=FR/ST=FR/O=SUSE/CN
-openssl req -new -key secrets/portus.key -out secrets/portus.csr -subj "/C=FR/ST=FR/O=SUSE/CN=portus-dr.suse-sap.net"
-openssl x509 -req -in secrets/portus.csr -CA secrets/rootca.crt -extfile extfile.cnf -CAkey secrets/rootca.key -CAcreateserial  -out secrets/portus.crt -days 500 -sha256
-----
 
-Make the servers aware of the new certificate:
+openssl req -new -key secrets/portus.key -out secrets/portus.csr \
+  -subj "/C=FR/ST=FR/O=SUSE/CN
+
+openssl req -new -key secrets/portus.key -out secrets/portus.csr \
+  -subj "/C=FR/ST=FR/O=SUSE/CN=portus-dr.suse-sap.net"
 
+openssl x509 -req -in secrets/portus.csr -CA secrets/rootca.crt \
+  -extfile extfile.cnf -CAkey secrets/rootca.key -CAcreateserial \
+  -out secrets/portus.crt -days 500 -sha256
+----
+
+. Make the servers aware of the new certificate:
++
 ----
 cp -p secrets/rootca.crt /etc/pki/trust/anchors/.net-ca.crt
 scp secrets/rootca.crt root@jumpbox.suse-sap.net:/etc/pki/trust/anchors/portus-dr.suse-sap.net-ca.crt
 ----
 
-Update the certificate on all servers that will need to interact with the Docker registry:
-
+. Update the certificate on all servers that will need to interact with the Docker registry:
++
 ----
 sudo update-ca-certificates
 sudo systemctl restart docker
 ----
 
-Start your Portus setup with the following command:
-
+. Start your Portus setup with the following command:
++
 ----
 docker-compose up -d
 ----
 
-Log in to Portus and set the registry as shown below.
-
+. Log in to Portus and set up the registry as shown below:
++
 image::portus-registry.png[title="Portus Registry",480,640,scaledwidth=80%,align=center]
 
-To install and configure a secure private registry using SUSE Linux Enterprise Server with the Container Module
-the needed componentes are Docker, registry and Portus.
-Create SSL certificates as needed. Distribute the CA certificate to all your kubernetes nodes.
+// FIXME: Does the above procedure continue here or is this something different?
```

Comment:
This is the main content issue where I was confused: Is this section a single procedure or are the above and below showing alternative procedures?

Reply:
this is one single procedure.

Reply:
Thanks! Fixed.